packages/admin-ui/src/DropdownMenu/components/DropdownMenuSeparator.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import { Separator, type SeparatorProps } from "~/Separator";
+import { makeDecoratable } from "~/utils";
+
+const DropdownMenuSeparatorBase = React.forwardRef<
+    React.ElementRef<typeof Separator>,
+    SeparatorProps
+>((props, ref) => {
+    return <Separator ref={ref} variant={"strong"} margin={"md"} {...props} />;
```

Comment:
I would remove the `ref` from the `Separator` component. What do you think?

Reply:
I can remove it, but I'm also like... I don't think it's that important / doesn't hurt.